sdk/tables/azure-data-tables/samples/tables_entity_operations.cpp

```diff
@@ -12,26 +13,19 @@ using namespace Azure::Data::Tables;
 using namespace Azure::Data::Tables::Models;
 const std::string TableName = "table";
 
-std::string GetConnectionString()
+// The following environment variables must be set before running the sample.
+// * ACCOUNT_NAME: The name of the storage account.
+std::string GetAccountName() { return std::getenv("ACCOUNT_NAME"); }
+std::string const GetServiceUrl()
 {
-  const static std::string ConnectionString = "";
-
-  if (!ConnectionString.empty())
-  {
-    return ConnectionString;
-  }
-  const static std::string envConnectionString = std::getenv("STANDARD_STORAGE_CONNECTION_STRING");
-  if (!envConnectionString.empty())
-  {
-    return envConnectionString;
-  }
-  throw std::runtime_error("Cannot find connection string.");
+  return std::string{"https://" + GetAccountName() + ".table.core.windows.net/"};
 }
 
 int main()
 {
-  auto tableServiceClient = TableServiceClient::CreateFromConnectionString(GetConnectionString());
-  auto tableClient = TableClient::CreateFromConnectionString(GetConnectionString(), TableName);
+  auto credential = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+  auto tableServiceClient = Azure::Data::Tables::TableServiceClient(GetServiceUrl(), credential);
+  auto tableClient = Azure::Data::Tables::TableClient(GetServiceUrl(), TableName, credential);
```

Comment:
And here, in the concrete sample, I think "using namespaces" for both Tables and Identity would be more practical, And then drop the full namespace qualifications here, so that the type names are shorter. When compiling and debugging, by interacting with the sample, users should be able to figure out, what comes from where.

Reply:
true